compiler/src/Language/Bond/Codegen/Cpp/Comm_cpp.hs

```diff
@@ -14,10 +14,10 @@ import Language.Bond.Codegen.TypeMapping
 import Language.Bond.Codegen.Util
 import qualified Language.Bond.Codegen.Cpp.Util as CPP
 
--- | Codegen template for generating /base_name/_comm_types.cpp containing
+-- | Codegen template for generating /base_name/_comm.cpp containing
```

Comment:
Yup. (Or stale. I don't remember whether gbc output comm_types.cpp in a prior incarnation or not...)